config/config.js

```diff
@@ -69,6 +69,33 @@ module.exports = {
       userCanEdit: true,
       adminOnly: false
     },
+    {
```

Comment:
I would move these to below `Allow IOC Deletion`, and you also need to add the names and descriptions to the Readme.md for documentation purposes.